### ISSUE TITLE
ComboBox has SelectionMode protected

### ISSUE THREAD

**Issue by mishun** [AvaloniaUI/Avalonia#4090]
Hi!
ComboBox is inherited from SelectingItemsControl where SelectionMode is declared as protected. As I could see, ListBox works around that like that:
```C#
        public new SelectionMode SelectionMode
        {
            get { return base.SelectionMode; }
            set { base.SelectionMode = value; }
        }
```
but that is not the case for ComboBox. Documentation at https://avaloniaui.net/docs/controls/combobox suggests that SelectionMode should be accessible. Is this a mistake in documentation or implementation?

**Comment by Gillibald** [comment]
Why would you want to change the SelectionMode?

**Comment by FoggyFinder** [comment]
> It is intended to be protected
> Single selection by design

https://gitter.im/AvaloniaUI/Avalonia?at=5f136a0c5ed1d537746b9927

**Comment by grokys** [comment]
Yeah `ComboBox` is intended to be single selection, so closing this as by-design.

**Comment by mishun** [comment]
Well, it most certainly is (although AlwaysSelected may make some sense, I guess), but the issue was about discrepancy between documentation under link and how code works.

**Comment by grokys** [comment]
Ah apologies! Will move this to the documentation website.
